src/context/Theme.tsx

```diff
@@ -368,9 +368,12 @@ const themeSettings = createTheme({
         },
         MuiCssBaseline: {
             styleOverrides: {
-                body: {
+                'body': {
                     minWidth: sm,
                 },
+                'body.loginPage': {
+                    minWidth: xs,
+                },
```

Comment:
Eventually we can remove this extra code but until we do a full "mobile audit" I think the smallest impact to the application will be only doing this on the login page.